gr-utils/modtool/templates/gr-newmod/docs/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@
 # Setup dependencies
 ########################################################################
 find_package(Doxygen)
+find_package(MathJax)
```

Comment:
This line needs to be removed. It will look for FindMathJax2.cmake which is not present in the OOT.

To find the package in the OOT, add the line:
```find_dependency(MathJax2)```
in cmake/Modules/GnuRadioConfig.cmake.in